src/VCAuthn/Controllers/WebhooksController.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using VCAuthn.IdentityServer.SessionStorage;
+
+namespace VCAuthn.Controllers
+{
+    [Route("/topic")]
+    public class WebhooksController : ControllerBase
+    {
+        private readonly ISessionStorageService _sessionStorageService;
+        private readonly ILogger<WebhooksController> _logger;
+
+        public WebhooksController(ISessionStorageService sessionStorageService, ILogger<WebhooksController> logger)
+        {
+            _sessionStorageService = sessionStorageService;
+            _logger = logger;
+        }
+        
+        
+        [HttpPost("{topic}")]
+        public async Task<ActionResult> GetTopicUpdate(string topic)
+        {
+            if (String.Equals("presentations", topic, StringComparison.InvariantCultureIgnoreCase) == false)
```

Comment:
I prefer a long soft in this case, because `!`sign easily hides between brackets,dots and other characters. Explicit ` == false` is way harder to miss, while reading.

Reply:
Fair enough :)